lib/index.js

```diff
@@ -121,6 +121,11 @@ export default function slackin({
 
     invite({ token, org, email, channel: chanId }, err => {
       if (err) {
+        if (err.message === `Already invited. Sending you to https://${org}.slack.com...`) {
+          return res
+          .status(403)
+          .json({ msg: err.message, redirectUrl: `https://${org}.slack.com` });
```

Comment:
Thanks for that @jpoon! You're totally right, a `403` didn't do what I thought it did. I think a `303` fits for this.

I'm not using `res.redirect` so I can give the user a quick message telling them they are being redirected.